scripts/printPkgPrNewPublishDirs.mts

```diff
@@ -0,0 +1,22 @@
+const publishDirs = {
+  '@mui/x-charts': 'packages/x-charts',
```

Comment:
When using `pnpm pack` we need to provide the package root instead of the build folder, because `pnpm pack` uses the `publishConfig` and fails if ran within `/build` subfolder.